scripts/gen-edpm-kustomize.sh

```diff
@@ -162,15 +162,14 @@ patches:
         edpm_ovn_dbs:
         - ${EDPM_OVN_DBS}
 
-        registry_name: quay.io
-        registry_namespace: podified-antelope-centos9
-        image_tag: current-podified
-        edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
-        edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
-        edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
-        edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
-        edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
-        edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+        registry_url: ${EDPM_REGISTRY_URL}
```

Comment:
You've to add these vars here too.

EDPM_REGISTRY_URL=${EDPM_REGISTRY_URL:"quay.io/podified-antelope-centos9"}
EDPM_CONTAINER_TAG=${EDPM_CONTAINER_TAG:"current-podified"}

Reply:
It is already defined in [Makefile](https://github.com/openstack-k8s-operators/install_yamls/pull/345/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R525), So we donot need to define it here.

Reply:
It is already defined in [Makefile](https://github.com/openstack-k8s-operators/install_yamls/pull/345/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R525), So we donot need to define it here.